.vib/grafana/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+const COMMAND_DELAY = 500;
+
+for (const command of ['click']) {
+    Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+        const origVal = originalFn(...args);
+
+        return new Promise((resolve) => {
+            setTimeout(() => {
+                resolve(origVal);
+            }, COMMAND_DELAY);
+        });
+    });
+}
+
+Cypress.Commands.add("login", (
+    username = Cypress.env("username"),
+    password = Cypress.env("password")
+) => {
+    cy.clearCookies();
+    cy.visit('/login')
+    cy.get('[name="user"]').type(username).should('have.value',username);
+    cy.get('input#current-password').type(password).should('have.value',password);
+    cy.get('[aria-label="Login button"]').click();
+    cy.get('div').contains('Invalid username').should('not.exist');
+});
```

Comment:
I personally would avoid using assertions in custom commands (unless they are meant to actually assert something, like `checkXXXX`). If a given functionality needs to be verified (in this case, the login), I'd better create a specific test case for that.

Reply:
Good point!
I've moved this assertion to the relevant test case.